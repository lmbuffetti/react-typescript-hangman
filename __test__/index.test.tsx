import * as React from 'react'
import IndexPage from '../pages/index'

describe('Pages', () => {
    describe('Index', () => {
        it('should pass', () => {
            expect(<IndexPage />).toMatchSnapshot();
        });
    })
})