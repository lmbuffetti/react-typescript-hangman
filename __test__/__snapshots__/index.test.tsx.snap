// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components 1`] = `
<div
  id="wrapper"
>
  <div
    id="loading"
  >
    <div
      id="loading-center"
    >
      <div
        id="loading-center-absolute"
      >
        <div
          className="object"
          id="object_four"
        />
        <div
          className="object"
          id="object_three"
        />
        <div
          className="object"
          id="object_two"
        />
        <div
          className="object"
          id="object_one"
        />
      </div>
    </div>
  </div>
  <div
    id="timer"
    style={
      {
        "width": "100%",
      }
    }
  />
  <svg
    height="300"
    width="300"
  >
    <g
      id="body"
      style={
        {
          "transform": "translate(0px,60px)",
        }
      }
    >
      <g
        id="head"
      >
        <circle
          cx="200"
          cy="80"
          fill="white"
          id="circleHead"
          r="20"
          stroke="black"
          strokeWidth="4"
        />
        <g
          id="rEyes"
        >
          <circle
            cx="193"
            cy="80"
            r="4"
          />
          <circle
            cx="207"
            cy="80"
            r="4"
          />
        </g>
      </g>
      <line
        x1="200"
        x2="200"
        y1="100"
        y2="150"
      />
      <line
        id="armL"
        x1="200"
        x2="170"
        y1="120"
        y2="140"
      />
      <line
        id="armR"
        x1="200"
        x2="230"
        y1="120"
        y2="140"
      />
      <line
        id="legL"
        x1="200"
        x2="180"
        y1="150"
        y2="190"
      />
      <line
        id="legR"
        x1="200"
        x2="220"
        y1="150"
        y2="190"
      />
    </g>
    <line
      x1="10"
      x2="150"
      y1="250"
      y2="250"
    />
    <line
      id="door1"
      x1="150"
      x2={200}
      y1="250"
      y2={250}
    />
    <line
      id="door2"
      x1={200}
      x2="250"
      y1={250}
      y2="250"
    />
    <line
      x1="250"
      x2="290"
      y1="250"
      y2="250"
    />
    <line
      x1="100"
      x2="100"
      y1="250"
      y2="20"
    />
    <line
      x1="100"
      x2="200"
      y1="20"
      y2="20"
    />
    <line
      id="rope"
      x1="200"
      x2="200"
      y1="20"
      y2="120"
    />
  </svg>
  <div
    id="displayWord"
  />
  <div
    id="wrapKeyboard"
  >
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      a
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      b
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      c
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      d
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      e
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      f
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      g
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      h
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      i
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      j
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      k
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      l
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      m
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      n
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      o
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      p
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      q
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      r
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      s
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      t
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      u
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      v
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      w
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      x
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      y
    </button>
    <button
      className="keyBoard correct"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      z
    </button>
  </div>
</div>
`;
